src/controller/postRouter.js

```diff
@@ -165,14 +156,6 @@ postRouter.patch("/:postId/likes", authenticate, postMiddleware.validateLike, as
     }
 });
 
-/**
- * Deletes a post by their id
- * Path Parameter
- *      :postId {string}
- * Response
- *      200 - Successfully deleted the post by their id
- *      400 - Post with id ${postId} not found
- */
 postRouter.delete("/:postId", postOwnerOrAdminAuthenticate, async (req, res) => {
```

Comment:
Route comments should stay. Kyle worked hard on them

Reply:
Huh, I do not know  why they got deleted. Yes I will return them.